contrib/seeds/README.md

```diff
@@ -8,7 +8,7 @@ and remove old versions as necessary.
 
 The seeds compiled into the release are created from sipa's DNS seed data, like this:
 
-    curl -s http://bitcoin.sipa.be/seeds.txt > seeds_main.txt
+    curl -s http://bitcoin.sipa.be/seeds.txt.gz | gunzip > seeds_main.txt
```

Comment:
@laanwj Good question! :-)  Perhaps it is a legacy thing, but for some reason I have it hard wired that `gzip -dc` is used when operating on stdin/stdout and `gunzip` is for unpacking `file.gz`, removing the original file and leaving `file`. They might be totally equivalent nowadays. If so, ignore my minor nit :-)